docs/gen_sections_docs

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+
+from pykickstart.parser import *
+from pykickstart.version import returnClassForVersion, DEVEL
+
+def _format_title(title, underline = '-'):
+    t = title.replace('|', ' or ')
+    return "\n%s\n%s\n" % (t, underline * len(t))
+
+if __name__ == "__main__":
+    chapter = 4
+    handler = returnClassForVersion(DEVEL)
+    for section_class in [PackageSection, PreScriptSection,
+                          PreInstallScriptSection, PostScriptSection,
+                          TracebackScriptSection, OnErrorScriptSection]:
```

Comment:
Done, although I'd prefer to see the sections moving under the commands handlers and become versioned as well.